---
title: "Population, evolutionary and ecological genomics"
layout: single
classes: wide
permalink: /
author_profile: true
date: 2021-03-04
header:
  overlay_color: "#C4C4C4"
  overlay_filter: "0.1"
  overlay_image: /assets/pics/banner.jpg
---

I study how plant populations adapt to their local environments, with emphasis on understanding the genetic and genomic architecture of beneficial traits. Currently I am a postdoc with [Peter Tiffin](http://www.cbs.umn.edu/tiffin/) at University of Minnesota, USA. I completed my PhD with [Outi Savolainen](https://www.oulu.fi/university/researcher/outi-savolainen) at University of Oulu, Finland.
<br>
## Genetic architecture of local adaptation
A key determinant of evolutionary responses is the genetic architecture of adaptive traits: i.e., the control of phenotypes by one or many loci, the interactions between alleles (e.g. pleiotropy, epistasis), and positions of loci in relation to genomic features (e.g. recombination rate, gene density).
<br>
<br>
As part of my PhD I studied how gene flow influences the genetic architecture of local adaptation, using closely adjacent population pairs of _Arabidopsis lyrata_ as my study subject. By reciprocally transplanting local and foreign genotypes, I first examined how local adaptation forms in the face of gene flow ([Hämälä et al. 2018](https://doi.org/10.1111/evo.13521)). I then utilized whole-genome sequence data, demography analyses, and genome scans for selection to elucidate genomic patterns underlying the adaptive differentiation ([Hämälä & Savolainen 2019](https://doi.org/10.1093/molbev/msz149)). Our study is among the first to empirically show that gene flow leads to clusters of adaptive loci in regions of low recombination. We also made a rare effort to distinguish between genetic trade-offs (antagonistic pleiotropy) and conditional neutrality at the level of individual loci.
<br>
{% include figure image_path="/assets/pics/gflow.png" caption="A combination of reciprocal transplant experiments, demography simulations, and genome scans for selection revealed that gene flow leads to clusters of adaptive loci in regions of low recombination." type=center %}
During my current postdoc appointment, I have continued to study the genetic architecture of local adaptation. I examined how selection on polygenic traits contributes to local adaptation in _Theobroma cacao_, the chocolate tree. Traditional methods for finding polygenic selection rely on predefined information; being either based on genotype-phenotype associations or gene groupings (e.g. biochemical pathways). In the absence of such information, I used expression data to group genes into functional modules, and used genomic data to search for signs of selection at the module level. Modules enriched for specific biological processes showed footprints of polygenic adaptation even though individual genes that were members of those groups were not subject to strong selection ([Hämälä et al. 2020a](https://doi.org/10.1093/molbev/msz206)). I further examined the role of pleiotropy (a single locus controlling multiple phenotypes) in regulating adaptive differentiation among populations of common ragweed (_Ambrosia artemisiifolia_). Classical theory predicts that pleiotropy constrains adaptation by increasing the chance that beneficial alleles also have deleterious effects. However, using empirical data and simulations, I found that when populations are far from their fitness optima, such as during a range expansion, pleiotropy becomes beneficial ([Hämälä et al. 2020b](https://doi.org/10.1371/journal.pgen.1008707)).
<br>
{% include figure image_path="/assets/pics/slim.png" caption="The fitness effects of pleiotropy depend on the distance from the selective optima." type=center %}
## Structural variant evolution in plants
My ongoing research focuses on understanding how genomic structural variants (SVs) influence evolutionary processes in natural populations. SVs can play important roles in adaptation and speciation, but our understanding of their overall fitness consequences is limited, with data and analyses primarily available for humans and short-lived domesticated species. Using a large collection of long-read based _de novo_ assemblies of _T. cacao_, I study how insertions, deletions, duplications, and inversions can both constrain and facilitate adaptation in this tropical tree species.
<br>
{% include figure image_path="/assets/pics/inversion.png" caption="Inversions accumulate derived, deleterious nucleotide variants as a result of suppressed recombination, thus constraining adaptation." type=center %}
